### DISCUSSION TITLE
No Signal via Display Port

### DISCUSSION THREAD

**Discussion by CHNtentes** [OE4T/tegra-demo-distro#237] [Q&A]
Hello.

I'm using `demo-image-full` in `tegra-demo-distro` with branch kirkstone. When I flash the image into SD and put it into `xavier-nx-devkit`, there is no video output via DP. I tried HDMI, and it worked fine.

The UART log after DP plugged in (Too long to paste directly below): [dp.txt](https://github.com/OE4T/tegra-demo-distro/files/10138780/dp.txt).

I see there was a similar post: [https://github.com/OE4T/meta-tegra/discussions/870](url). Is my issue the same with this one?

Thanks a lot.

**Comment by madisongh** [comment]
Have you looked at [this section](https://docs.nvidia.com/jetson/archives/r35.1/DeveloperGuide/text/SD/Kernel/DisplayConfigurationAndBringUp/XavierDisplayconfig.html) of the NVIDIA docs?  For some DP monitors you may have to tweak the display configuration in the device tree to get it to work.
